src/utils/copyToClipboard.js

```diff
@@ -0,0 +1,8 @@
+import { toast } from "react-hot-toast";
+
+export const handleCopy = (dataToCopy) => {
+  navigator.clipboard.writeText(dataToCopy).then(
+    () => toast.success(`Copied ${dataToCopy} to clipboard!`),
+    () => toast.error("Failed to copy code to clipboard."),
```

Comment:
General Feedback: The utility function shouldn't handle showing the toast notification, because utility functions should be reusable pieces of code. Not every time we will show a notification when something is copied to clipboard.

Also, the naming - instead of `handleCopy`, we can rename it to `copyToClipboard`